### PR TITLE
Android-ndk fix

### LINKED ISSUE #265: Cannot install android-ndk
When I try to install android-ndk with
```
umake android android-ndk
```
I get:
```
ERROR: The checksum of http://dl.google.com/android/repository/android-ndk-r11-linux-x86_64.zip doesn't match. Corrupted download? Aborting.
```

### PR DESCRIPTION
**Pull request by LyzardKing**
This reflects the changes made on the download page for the android ndk.
The download is now a zipfile, only for 64 bit arch.
The checksum now is sha1 instead of md5.
The large test works on my system.
The medium tests are fixed.
Since I know nothing about the ndk, I'm not sure what to test.
At the moment the only check is the existence of the main executable, as described on the webpage.
If there is something better that needs to be tested please let me know.

Fixes #265

### CODE REVIEW

**Comment by LyzardKing** [comment]
I also added clang as a dependency, since it is now the supported compiler.

File: tests/medium/test_android.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ class AndroidNDKContainer(ContainerTests, test_android.AndroidNDKTests):
 
     def setUp(self):
         self.hosts = {443: ["developer.android.com"]}
-        self.apt_repo_override_path = os.path.join(self.APT_FAKE_REPO_PATH, 'android')
+        self.apt_repo_override_path = os.path.join(self.APT_FAKE_REPO_PATH, 'android-ndk')
```
Comment (didrocks) [inline tests/medium/test_android.py]: I don't see you creating a separate android-ndk repository for medium tests but you did change that path, did you get the medium tests working with this change? Was there already a separate repository that I don't see? :)

**Comment by didrocks** [comment]
And here is a review! Just a comment on the medium repository "android-ndk" that you now reference. I'm quite surprised this works for you or maybe you forgot to include in the commits a "docker/create_packages.sh" changes?

Other than that, I did test it and it's good.

**Comment by LyzardKing** [comment]
Yes, I forgot about the docker/create_packages.sh file.
I read that gcc is deprecated in the ndk (https://android.googlesource.com/platform/ndk.git/+/master/CHANGELOG.md)
Should we add clang to the normal android repo in create_packages?

**Comment by didrocks** [comment]
yeah, let's follow upstream's practices.

**Comment by LyzardKing** [comment]
I noticed that the zipfile is in another directory on the google server.
I'll change the download page in our test.

**Comment by didrocks** [comment]
Perfect! I'm merging this :)

Once the other is merged as well, can you just sum up what tests are passing and not passing for you? I must say I'm a little bit lost in the email threads :) (I know that you can't run the tests in the vm due to python2 exceptions)

**Pull request merged**